apps/coursition/src/app/actions.ts

```diff
@@ -45,8 +49,91 @@ export const createUser = async ({ email, password }: { email: string; password:
     })
 }
 
-export const getInTouch = async (formdata: FormData) => {
-  const data = Object.fromEntries(formdata)
-  console.log({ data })
-  // todo: call resend helper
+export const getInTouch = async (prevState: { message: string }, formdata: FormData) => {
+  const data = Object.fromEntries(formdata) as Record<string, string>
+
+  const { email = '', firstName, lastName, comment } = data
+  return await send({
+    react: Components.GetInTouchEmailTemplate({ email, fullName: `${firstName} ${lastName}`, comment }),
+    from: 'Acme <onboarding@resend.dev>',
+    to: [''], // company's email
+    subject: 'Get In Touch',
+  })
+    .then(() => ({ message: 'Your invitation has been sent' }))
+    .catch(() => ({ message: "Sorry, your invitation wasn't sent" }))
+}
+
+export const sendResetPassword = async (
+  _: { step: number; error?: string | null },
+  formdata: FormData,
+): Promise<{ step: number; error?: string }> => {
+  try {
+    const email = formdata.get('email') as string
+
+    const user = await prisma.user.findFirst({ where: { email } })
+
+    if (!user?.id) return { error: 'user not found', step: 1 }
+
+    const record = (await prisma.passwordReset
```

Comment:
**nitpick:** Again combination of `then/catch` with await. We should be consistent and do only `async/await`.